app/views/media/show_albums.html.erb

```diff
@@ -0,0 +1 @@
+<%= render partial: 'show_albums' %>
```

Comment:
I think if our view is just going to say to render a partial with the same name we can skip having this view altogether, and wherever we would have rendered this view, we render the partial instead.